flows/runs/run.go

```diff
@@ -549,11 +541,6 @@ func (r *flowRun) MarshalJSON() ([]byte, error) {
 		re.Child = r.child.UUID()
 	}
 
-	re.Input, err = utils.EnvelopeFromTyped(r.input)
```

Comment:
Can't load the input here because we haven't yet loaded the channels. We don't actually need the input since we should have new input if we're reloading runs. Anyway in the next pr I'm reworking how we load stuff so it's easier to resolve references to other objects.